.github/workflows/test.yml

```diff
@@ -20,9 +20,23 @@ jobs:
           echo "RUST_STABLE=$rust_stable" | tee -a $GITHUB_ENV
 
       - if: runner.os == 'Linux'
+        env:
+          PB_REL: https://github.com/protocolbuffers/protobuf/releases
+          PROTOC_VERSION: '22.0'
         run: |
           sudo apt-get update
-          sudo apt-get install -y libudev-dev libssl-dev libsasl2-dev libzstd-dev
+          sudo apt-get install -y \
+            curl \
+            libudev-dev \
+            libsasl2-dev \
+            libssl-dev \
+            libzstd-dev \
+            unzip
+          echo 'installing protoc'
+          curl -LO $PB_REL/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip
+          unzip protoc-$PROTOC_VERSION-linux-x86_64.zip -d $HOME/.local
```

Comment:
Not a fan of installing unsigned binaries. Also presumably has to be done in the other ci file? It's inexcusable that the protobuf devs have not released updates deb packages for Ubuntu or Debian for LTS or stable.

Perhaps we can snag the testing deb packages from Debian? Can also check their signatures

Reply:
Agreed